tests/unit/action-toolbar-button.spec.ts

```diff
@@ -71,4 +70,106 @@ describe('ActionToolbar', () => {
     actionsWrappers.at(1).trigger('click');
     expect(wrapper.emitted().actionClicked[1]).toEqual(['unpivot']);
   });
+
+  describe('When clicking on the "To lowercase" operation', () => {
+    it('should close any open step form', async () => {
+      const store: Store<any> = setupMockStore({
+        pipeline: [{ name: 'domain', domain: 'myDomain' }],
+        selectedColumns: ['foo'],
+      });
+      const wrapper = mount(ActionToolbarButton, {
+        propsData: { category: 'text' },
+        store,
+        localVue,
+      });
+      const actionsWrappers = wrapper.findAll('.action-menu__option');
+      await actionsWrappers.at(0).trigger('click');
+      expect(store.state.vqb.currentStepFormName).toEqual(undefined);
+    });
+
+    it('should insert a lowercase step in pipeline', async () => {
+      const store: Store<any> = setupMockStore({
+        pipeline: [{ name: 'domain', domain: 'myDomain' }],
+        selectedColumns: ['foo'],
+      });
+      const wrapper = mount(ActionToolbarButton, {
+        propsData: { category: 'text' },
+        store,
+        localVue,
+      });
+      const actionsWrappers = wrapper.findAll('.action-menu__option');
+      await actionsWrappers.at(0).trigger('click');
+      expect(store.state.vqb.pipeline).toEqual([
+        { name: 'domain', domain: 'myDomain' },
+        { name: 'lowercase', column: 'foo' },
+      ]);
+      expect(store.state.vqb.selectedStepIndex).toEqual(1);
+    });
+
+    it('should emit a close event', async () => {
+      const store: Store<any> = setupMockStore({
+        pipeline: [{ name: 'domain', domain: 'myDomain' }],
+        selectedColumns: ['foo'],
+      });
+      const wrapper = mount(ActionToolbarButton, {
+        propsData: { category: 'text' },
+        store,
+        localVue,
+      });
+      const actionsWrappers = wrapper.findAll('.action-menu__option');
+      await actionsWrappers.at(0).trigger('click');
+      expect(wrapper.emitted().closed).toBeTruthy();
+    });
+  });
+
+  describe('When clicking on the "To uppercase" operation', () => {
+    it('should close any open step form', async () => {
+      const store: Store<any> = setupMockStore({
```

Comment:
Do you really need to specify the `Store<any>` type annotation?

Reply:
yes to make typescript happy :)
not sure to find something more elegant quickly... should we merge? :)